docs/docs/user/publishing.md

```diff
@@ -0,0 +1,155 @@
+# Publishing
+
+Simplest way to start publishing to Hermes is sending POST request on topic endpoint::
+
+```
+curl -X POST -H "Content-Type: application/json" http://hermes-frontend/topics/group.topic -d '{"message": "Hello world!"}'
+```
+
+## Creating group
+
+As [data model](/overview/data-model) describes, topics are gathered into topic groups. If you don't have a group yet,
+you need to create one by sending POST with `application/json` content type at
+
+```
+/groups
+```
+
+Body of request must contain:
+
+* name: group name
+* supportTeam: team that should have administrative access to this group
+
+Minimal request:
+
+```json
+{"groupName": "my-group", "supportTeam": "My Team"}
+```
+
+
+## Creating topic
+
+Use Hermes Management REST API to create subscription by sending POST request with `application/json` content type
+on topics resource:
+
+```
+/topics
+```
+
+Body of request must contain at least:
+
+* name: fully qualified name of topic including group name, separated with a dot (see: [naming convention](/overview/data-model#naming-convention))
+
+Minimal request:
+
+```json
+{"name": "my-group.my-topic"}
+```
+
+All options:
+
+Option            | Description                             | Options     | Default value
+---------------   | --------------------------------------- | ----------- | -------------
+ack               | acknowledgement level                   | ALL, LEADER | LEADER
+retentionTime     | time to keep data in Kafka in days      | -           | 1
+trackingEnabled   | track incoming messages?                | -           | false
+contentType       | format of data sent to Kafka            | AVRO, JSON  | JSON
+validationEnabled | use message schema to validate messages | -           | false for JSON, true for AVRO
+
+Request that specifies all available options:
+
+```
+{
+    "name": "myTopic",
+    "description": "This is my topic",
+    "ack": "LEADER",
+    "retentionTime": 1,
+    "trackingEnabled": false,
+    "contentType": "JSON",
+    "validationEnabled": false
+}
+```
+
+
+## Message format
+
+Each topic has defined content type. This describes what will be the format of data sent to Kafka.
+At this point Hermes supports messages sent in:
+
+* [JSON](publishing-json)
+* [Avro](publishing-avro)
+
+Read provided documentation for detailed description of features and mechanisms for each format.
+
+### Message schema
+
+When enabling validation (or using AVRO content type), message schema needs to be attached before sending first message.
```

Comment:
In one place we use uppercases for word avro in other lowercases. I think it would be nice to use consistent way.